framework/src/problems/EigenProblem.C

```diff
@@ -361,7 +361,7 @@ EigenProblem::solve()
 void
 EigenProblem::init()
 {
-#if !PETSC_RELEASE_LESS_THAN(3, 13, 0)
+#if !PETSC_RELEASE_LESS_THAN(3, 13, 0) && LIBMESH_HAVE_SLEPC
```

Comment:
@fdkong I had to add this because if libmesh doesn't have SLEPc then the `NonlinearEigenSystem` class doesn't have a `sys()` method

Reply:
Good for me